core/templates/services/UpgradedServices.ts

```diff
@@ -727,6 +730,9 @@ export class UpgradedServices {
       new ɵangular_packages_common_http_http_d();
 
     // Topological level: 1.
+    upgradedServices['AlgebraicExpressionInputValidationService'] =
```

Comment:
@iamprayush please add this service to `oppia-angular-root.component.ts`. The steps for doing this is mentioned in the point 12 here: https://github.com/oppia/oppia/wiki/Angular-Migration#implementation-details-to-migrate-services

Reply:
Done.